src/renderers/WebGLRenderer.js

```diff
@@ -2820,6 +2820,8 @@ function WebGLRenderer( parameters ) {
 
 		}
 
+		if (dstTexture.generateMipmaps) _gl.generateMipmap( _gl.TEXTURE_2D );
```

Comment:
I'm not sure this check is safe. What if the flag is set to `false` since mipmaps were defined manually?